wiki/Performance_points/ppv2/th.md

```diff
@@ -0,0 +1,8 @@
+---
+stub: true
+tags:
+  - ppv2
+---
+
+# ppv2
```

Comment:
Add a page break between `ppv2` and the text block below it to respect a formatting conformity between the articles.

Reply:
okie

Reply:
In fact I was not clear enough in my explanations:

To pass the check of the file, you must put a space between line 7 and line 8, otherwise the file cannot be merged.